### PR TITLE
Update Aspects Course Comparison Dashboard feature release notes

### PR DESCRIPTION
**Pull request by crathbun428**
Updated feature release notes, added image files, fixed inline links.

### CODE REVIEW

File: source/community/release_notes/sumac/aspects_comp_dashboard.rst [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+In this new dashboard, users can view metrics for all runs of a course combined or broken out by each individual run by switching between two tabs at the top of the dashboard. Those who want to compare all runs of a course at a glance can easily do so by filtering the dashboard by course and viewing the Run Metrics tab.
+
+.. image:: /_images/sumac-aspects-run-metrics-1
+
+Spot high and low enrollment and activity
+=========================================
+
+The Course Comparison Dashboard makes it easy to identify courses with the highest and lowest levels of enrollment and activity with tables that allow users to sort courses or runs by current enrollment or active learner count. 
+
+.. image:: /_images/sumac-aspects-course-info-2
```
Comment (sarina) [inline source/community/release_notes/sumac/aspects_comp_dashboard.rst:22]: You have to put `.png` at the end of these filenames! (full file names, not just title). Also the path to the folder is wrong.

```suggestion
.. image:: /_images/release_notes/sumac/sumac-aspects-course-info-2.png
```

Comment (sarina) [reply source/community/release_notes/sumac/aspects_comp_dashboard.rst:22]: Be sure to fix all the filenames in this doc please :)

**Comment by sarina** [comment]
Looks good to me! Check out the rendered view (https://docsopenedxorg--642.org.readthedocs.build/en/642/community/release_notes/sumac/aspects_comp_dashboard.html) to make sure you're happy with it, and I'll merge.

**Comment by crathbun428** [comment]
@sarina - thank you for your review. I added one return between the final picture and final paragraph. Rest looks good to me.

**Pull request merged**